projects/plugins/boost/app/assets/src/js/utils/make-admin-ajax-request.ts

```diff
@@ -22,7 +22,7 @@ export async function makeAdminAjaxRequest< T = JSONObject >( payload: JSONObjec
 		method: 'post',
 		body: new URLSearchParams( {
 			...payload,
-			...{ nonce: Jetpack_Boost.criticalCssAjaxNonce },
+			...{ nonce: Jetpack_Boost.criticalCssDismissRecommendationsNonce },
```

Comment:
It's probably a better idea to not get the nonce directly in `makeAdminAjaxRequest`. The reason is: `makeAdminAjaxRequest` can be used for other purposes than dismiss recommendations. So, my suggestion is to remove line 25 and send nonce directly with payload. It's a little bit of a hassle but there is no other option if we want to reuse the function.

In PR #20886, I used this function to dismiss a prompt. See https://github.com/Automattic/jetpack/pull/20886#issuecomment-923802666

Reply:
Any thoughts on this? @thingalon

Reply:
This is a good point; it will probably need to be a parameter to `makeAdminRequest`.

Reply:
Nonce will be coming as a part of the payload.